backend/tests/endpoints/conftest.py

```diff
@@ -79,3 +79,47 @@ def fixture_mock_dataStorage_delete(mocker):
     zc_core_update_data = AsyncMock()
     mocker.patch("utils.db.DataStorage.delete", side_effect=zc_core_update_data)
     return zc_core_update_data
+
+
+@pytest.fixture(name="init_fake_room")
+def fixture_initialize_fake_room_data():
+    """A fixture that initialises a new fake room data for each test
+
+    Returns:
+        Dict: An room dictionary
+    """
+    room_data = {
```

Comment:
You can place a return here since you're simply returning the data for the room
```suggestion
    return {
```